Microsoft.Toolkit.Uwp.UI.Animations/Behaviors/FadeHeaderBehavior.cs

```diff
@@ -91,56 +89,44 @@ public UIElement HeaderElement
         }
 
         /// <summary>
-        /// Called when the associated object is loaded.
-        /// </summary>
-        /// <param name="sender">The associated object</param>
-        /// <param name="routedEventArgs">The <see cref="RoutedEventArgs"/> instance containing the event data</param>
-        private void AssociatedObjectOnLoaded(object sender, RoutedEventArgs routedEventArgs)
-        {
-            AssignFadeAnimation();
-        }
-
-        /// <summary>
         /// Uses Composition API to get the UIElement and sets an ExpressionAnimation
         /// The ExpressionAnimation uses the height of the UIElement to calculate an opacity value
         /// for the Header as it is scrolling off-screen. The opacity reaches 0 when the Header
         /// is entirely scrolled off.
         /// </summary>
-        private void AssignFadeAnimation()
+        private bool AssignFadeAnimation()
         {
             // Confirm that Windows.UI.Xaml.Hosting.ElementCompositionPreview is available (Windows 10 10586 or later).
             if (!ApiInformation.IsMethodPresent("Windows.UI.Xaml.Hosting.ElementCompositionPreview", nameof(ElementCompositionPreview.GetScrollViewerManipulationPropertySet)))
             {
-                return;
+                return false;
```

Comment:
I think it would be better to return "true" here, otherwise the method will be called again and again on each SizeChanged.